From e1c04bcd5e4bbe1f5567cd8a44fe4ef5ea5c57dd Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 24 Apr 2018 11:11:31 +0800
Subject: [PATCH 3687/5242] MLK-18123-1 gpu: imx: imx8_prg: Rename
 prg_put_auxilary() to prg_set_primary()

commit  53ebc262823d6f3ee39024fb9144c1702aa7bb98 from
https://source.codeaurora.org/external/imx/linux-imx.git

A cosmetic change to rename prg_put_auxilary() to prg_set_primary().

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/imx8_dprc.c   |    2 +-
 drivers/gpu/imx/imx8_prg.c    |    4 ++--
 include/video/imx8-prefetch.h |    2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/imx/imx8_dprc.c b/drivers/gpu/imx/imx8_dprc.c
index f8ab82a..bd32c31 100644
--- a/drivers/gpu/imx/imx8_dprc.c
+++ b/drivers/gpu/imx/imx8_dprc.c
@@ -428,7 +428,7 @@ void dprc_configure(struct dprc *dprc, unsigned int stream_id,
 		if (dprc->sc_resource == SC_R_DC_0_BLIT0
 			&& dprc->devtype->has_fixup) {
 			dprc_prg_sel_configure(dprc, SC_R_DC_0_BLIT0, 0);
-			prg_put_auxiliary(dprc->prgs[0]);
+			prg_set_primary(dprc->prgs[0]);
 		}
 
 		switch (modifier) {
diff --git a/drivers/gpu/imx/imx8_prg.c b/drivers/gpu/imx/imx8_prg.c
index f1db1c1..bee9476 100644
--- a/drivers/gpu/imx/imx8_prg.c
+++ b/drivers/gpu/imx/imx8_prg.c
@@ -303,14 +303,14 @@ void prg_set_auxiliary(struct prg *prg)
 }
 EXPORT_SYMBOL_GPL(prg_set_auxiliary);
 
-void prg_put_auxiliary(struct prg *prg)
+void prg_set_primary(struct prg *prg)
 {
 	if (WARN_ON(!prg))
 		return;
 
 	prg->is_auxiliary = false;
 }
-EXPORT_SYMBOL_GPL(prg_put_auxiliary);
+EXPORT_SYMBOL_GPL(prg_set_primary);
 
 void prg_set_blit(struct prg *prg)
 {
diff --git a/include/video/imx8-prefetch.h b/include/video/imx8-prefetch.h
index aa1bd1c..c89219d 100644
--- a/include/video/imx8-prefetch.h
+++ b/include/video/imx8-prefetch.h
@@ -42,7 +42,7 @@ void prg_configure(struct prg *prg, unsigned int width, unsigned int height,
 bool prg_stride_double_check(struct prg *prg,
 			     unsigned int stride, dma_addr_t baddr);
 void prg_set_auxiliary(struct prg *prg);
-void prg_put_auxiliary(struct prg *prg);
+void prg_set_primary(struct prg *prg);
 void prg_set_blit(struct prg *prg);
 
 struct dprc;
-- 
1.7.9.5

